index.html

```diff
@@ -9,10 +9,10 @@
 <div class="main-wrapper">
 	<header>
 		<h1 class="blue-underline top-margin-fix">My name is Josh,<br>I make Web Applications</h1>
-		<img class="selfie" src="images/selfie.jpg">
+		<img class="selfie" alt="Image of Joshua Lloyd" src="images/selfie.jpg">
 	</header>
-	<article>
-		<section><h2 class="blue-underline">Bio</h2><p>I have always had an interest in technology and design. I graduated from MTSU in 2006 with a degree in graphic design. While I started doing work mostly in print, over the years I have picked up more and more knowledge and interest in web design and development. Finally in the spring of 2017, after being frustrated in the pace of my skill development. I made the decision to pivot to dive deeply into full-stack web development. I particulary am interested in using javascript everywhere since it has become such a widely implemented programing interface for a number of technologies.</p></section>
+		<section><h2 class="blue-underline">Bio</h2><p>I have always had an interest in technology and design. I graduated from MTSU in 2006 with a degree in graphic design. While I started doing work mostly in print, over the years I have picked up more and more knowledge and interest in web design and development. Finally in the spring of 2017, after being frustrated in the pace of my skill development. I made the decision to pivot to dive deeply into full-stack web development. I particulary am interested in using javascript everywhere since it has become such a widely implemented programing interface for a number of technologies.</p>
```

Comment:
This does not need to be indented as it is a sibling element of the header element.